ui/admin/app/templates/scopes/scope/workers/index.hbs

```diff
@@ -34,64 +34,46 @@
-              as |filter|
-            }}
-              {{#if filter.allowedValues}}
-                <Rose::Dropdown
-                  @text={{t (concat 'form.' filter.name '.label')}}
-                  @count={{filter.selectedValue.length}}
-                  @ignoreClickInside={{true}}
-                  as |dropdown|
-                >
-                  {{#each-in
-                    (group-by 'key' filter.allowedValues)
-                    as |key tags|
-                  }}
-                    <dropdown.section @title={{key}} @icon='tag' />
-                    <form.checkboxGroup
-                      @name={{key}}
-                      @items={{tags}}
-                      @selectedItems={{filter.selectedValue}}
-                      @onChange={{fn this.callFilterBy filter.name}}
-                      @itemEqualityFunc={{this.isEqual}}
-                      as |group|
-                    >
-                      <dropdown.item class='indent-label-1'>
-                        <group.checkbox
-                          @label={{group.item.value}}
-                          value={{group.item}}
-                        />
-                      </dropdown.item>
-                    </form.checkboxGroup>
-                  {{/each-in}}
-                </Rose::Dropdown>
-              {{/if}}
-            {{/let}}
-            {{#if
-              (has-resource-filter-selections
-                'scopes.scope.workers.index' 'tags'
-              )
-            }}
-              <Hds::Button
-                @size='small'
-                @icon='x'
-                @text={{t 'actions.clear-all-filters'}}
-                {{on 'click' this.callClearAllFilters}}
-              />
-            {{/if}}
-          </Rose::Form>
-        </toolbar.primary>
-
-        <toolbar.secondary>
-          <ToolbarRefresher @onClick={{this.refresh}} />
-        </toolbar.secondary>
 
-      </Rose::Toolbar>
+      <div class='workers-filtering'>
+        <Dropdown
+          name='tags'
+          @listPosition='bottom-left'
+          @toggleText={{t 'form.tags.label'}}
+          @itemOptions={{this.groupedTags}}
+          @checkedItems={{this.tags}}
+          @applyFilter={{fn this.applyFilter 'tags'}}
+          @isGrouped={{true}}
+          as |FD selectItem itemOptions|
+        >
+          {{#each-in (group-by 'key' (decode itemOptions)) as |key groupItems|}}
+            <FD.Generic>
+              <Hds::Icon @name='tag' @isInline={{true}} />
+              <Hds::Text::Display
+                @size='100'
+                @weight='medium'
+              >{{key}}</Hds::Text::Display>
+            </FD.Generic>
+            {{#each groupItems as |item|}}
+              <FD.Checkbox
+                @value={{encode item}}
+                checked={{includes (encode item) this.tags}}
+                {{on 'change' selectItem}}
+              >
+                {{item.value}}
```

Comment:
I think you could have just used the object you have from `@itemOptions` directly without having to re-encode here